.gitignore

```diff
@@ -9,3 +9,4 @@ __pycache__/
 
 /.clangd/
 /compile_commands.json
+TAGS
```

Comment:
What is TAGS and why is it in the global .gitignore?

Reply:
Apologies; this PR includes commits I did not intend to (including the one that added `TAGS` to this file). Will update.